cli/cobra.go

```diff
@@ -102,7 +102,7 @@ func managementSubCommands(cmd *cobra.Command) []*cobra.Command {
 var usageTemplate = `Usage:
 
 {{- if not .HasSubCommands}}	{{.UseLine}}{{end}}
-{{- if .HasSubCommands}}	{{ .CommandPath}} COMMAND{{end}}
+{{- if .HasSubCommands}}	{{ .CommandPath}}{{- if .HasAvailableFlags}} [OPTIONS]{{end}} COMMAND{{end}}
```

Comment:
nit: Isn't this change unrelated ? :stuck_out_tongue_closed_eyes:

Reply:
The `docker stack` command itself now has options, but there was no `[OPTIONS]` printed 😞